.github/workflows/build_poc.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Push Docker Image
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-push:
+    env:
+      GIT_DEPTH: "0"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
```

Comment:
There is already v3, consider bumping it.

Reply:
bumped